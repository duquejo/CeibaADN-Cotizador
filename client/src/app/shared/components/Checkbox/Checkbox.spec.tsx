import * as React from 'react';
import { render, mount, ShallowWrapper, shallow } from 'enzyme';
import { CheckBox } from './index';


describe('Pruebas sobre componente <CheckBox />', () => {

    let componentWrapper: ShallowWrapper;

    beforeEach( () => {
        componentWrapper = shallow(<CheckBox/>);
    });

    it('DeberÃ­a renderizar el elemento', () => {
        expect( componentWrapper ).toMatchSnapshot();
    });
});