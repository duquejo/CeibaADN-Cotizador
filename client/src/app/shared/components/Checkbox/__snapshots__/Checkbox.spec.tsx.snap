// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas sobre componente <CheckBox /> Deber√≠a renderizar el elemento 1`] = `
<input
  className="sc-bdvvtL jbOFTi"
  type="checkbox"
/>
`;
